clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -2033,6 +2033,31 @@ def MemCpyOp : CIR_Op<"libc.memcpy"> {
   }];
 }
 
+//===----------------------------------------------------------------------===//
+// FAbsOp
+//===----------------------------------------------------------------------===//
+
+def FAbsOp : CIR_Op<"libc.fabs", [Pure, SameOperandsAndResultType]> {
+  let arguments = (ins AnyFloat:$src);
+  let results = (outs AnyFloat:$result);
+  let summary = "Equivalent to libc's `fabs`";
```

Comment:
The summary could be your description first line `Returns absolute value for floating-point input`. And then...